File: app-geekhome/src/main/java/eu/automateeverything/rest/ResourceNotFoundExceptionMapper.java
package eu.automateeverything.rest;

import javax.ws.rs.core.Response;
import javax.ws.rs.ext.ExceptionMapper;

public class ResourceNotFoundExceptionMapper implements ExceptionMapper<ResourceNotFoundException> {

    @Override
    public Response toResponse(ResourceNotFoundException ex) {
        return Response
                .status(404)
                .build();
    }
}

File: app-server/src/main/java/eu/automateeverything/rest/ResourceNotFoundExceptionMapper.kt
package eu.automateeverything.rest

import jakarta.ws.rs.core.Response
import jakarta.ws.rs.ext.ExceptionMapper

class ResourceNotFoundExceptionMapper : ExceptionMapper<ResourceNotFoundException?> {
    override fun toResponse(ex: ResourceNotFoundException?): Response {
        return Response
            .status(404)
            .build()
    }
}